docs/src/rsh/compute/index.md

```diff
@@ -1836,6 +1836,8 @@ load: /examples/realtime/index.rsh
 range: 12 - 18
 ```
 
+Notice how `{!rsh} lastConsensusTime` is called on line 17 after the publication on line 12.
```

Comment:
Please see [PR #993](https://github.com/reach-sh/reach-lang/pull/993/files/4ed2cf7c3f9dac6d9b75136f5bb62bb0b36285da)

For an example of an intuitive explanation of the sample. 

If you want to talk about the code samples please schedule a call with me.